bigstring.opam

```diff
@@ -13,9 +13,11 @@ build: [
   [ "dune" "runtest" "-p" name "-j" jobs ] {with-test}
 ]
 depends: [
-  "dune" {>= "1.0"}
+  "dune" {>= "1.2"}
   "base-bigarray"
   "base-bytes"
   "ocaml" {>= "4.03.0"}
+  "alcotest" {with-test}
+  "bigstring-unix" {with-test}
```

Comment:
this seems unecessary? tests for bigstring-unix should be in the bigstring-unix package :-)

Reply:
Should I try to disable tests for ``bigstring`` ? But why care?